harvest/collator.py

```diff
@@ -38,6 +40,14 @@ def __init__(self, repo_url, creds, branch, repo_path=None, validate=True):
         self.git_repo = None
         self.validate = validate
 
+    @property
+    def local_path(self):
+        """Provide the local OS path to the Git repo."""
+        if self.repo_path:
+            return self.repo_path
+        tmpdir = PurePath(tempfile.gettempdir())
+        return str(tmpdir.joinpath('harvest', self.org, self.repo))
```

Comment:
Shouldn't we set this once it is calculated?

```
self.repo_path = str(tmpdir.joinpath('harvest', self.org, self.repo))
return self.repo_path
```

Reply:
No we don't want to do that.

Reply:
repo_path can be passed on the command line.  If it is then we want that to be the local_path property, if it isn't then we want to calculate the local path property but we don't want to set the repo_path attribute to that calculation because it would cause undesirable behavior in the checkout method.  I'm actually just adding this property for use in a custom report I want to write.  It isn't explicitly needed by harvest and was calculated previously as a variable in the checkout method.

Reply:
OK! Thanks for the explanation.